guides/micronaut-data-mongodb-synchronous/micronaut-data-mongodb-synchronous.adoc

```diff
@@ -0,0 +1,129 @@
+common:header.adoc[]
+
+You will use https://www.mongodb.com[MongoDB] for persistence.
+
+common:requirements.adoc[]
+
+common:completesolution.adoc[]
+
+common:create-app-features.adoc[]
+
+=== POJO
+
+Create a `Fruit` POJO:
+
+source:Fruit[]
+
+callout:mapped-entity[1]
+callout:mapped-entity-id[2]
+callout:constraints[3]
+
+=== Repository
+
+Create a repository interface to encapsulate the CRUD actions for `Fruit`.
+
+source:FruitRepository[]
+
+<1> Annotate with `@MongoRepository`.
+<2> Add a finder for finding fruit by a list of names.
+
+=== Service
+
+Create a service `FruitService` to interact with the `Fruit` repository.
```

Comment:
I don't see the value in a service layer for this guide. It would make sense if the service was transactional or did other business logic, but this is just a light wrapper around the repository methods.

Reply:
Ok, after a discussion with @sdelamo we decided to change the service to an interface and supply a default implementation.  This enabled us to demonstrate replacement under test for checking the controller without going through the database.  I added a new test for this, and referenced it in the guide.

It's a little contrived, but we thought showing "good process" was better than showing the quickest route.

It's a shame `@ExecuteOn` doesn't work on Services, as it would be good to mark this service as requiring a blockable pool, so if another controller was added to use it, the implementer would not need to know that `ExecuteOn` is a requirement